packages/caip/src/adapters/banxa/index.ts

```diff
@@ -57,9 +57,9 @@ export const getSupportedBanxaAssets = () =>
  * chain won't be exactly the same as ours.
  */
 const chainIdToBanxaBlockchainCodeMap: Record<ChainId, string> = {
-  'eip155:1': 'ETH',
-  'bip122:000000000019d6689c085ae165831e93': 'BTC',
-  'cosmos:cosmoshub-4': 'COSMOS'
+  ['eip155:1' as ChainId]: 'ETH',
+  ['bip122:000000000019d6689c085ae165831e93' as ChainId]: 'BTC',
+  ['cosmos:cosmoshub-4' as ChainId]: 'COSMOS'
```

Comment:
These type-casted string literals will go away once we properly implement `SUPPORTED_CHAIN_IDS` or similar.

Example implementation in https://github.com/shapeshift/lib/pull/688/commits/a26d0315119e7d7527476890000b529008bd4248#diff-bc09084a5595dff54122b864f8a3718cf0403c148a7b6b667452b710bdaaf8b5